SUPPORTED_MODELS.md

```diff
@@ -263,7 +263,7 @@ The pipeline can work with other similar topologies produced by `optimum-intel`
       <td>Supported</td>
       <td>Supported</td>
       <td>Supported</td>
-      <td>Not supported</td>
+      <td>Partially supported</td>
```

Comment:
@yatarkan, FYI, it seems bool isn't enough for https://yatarkan.github.io/openvino.genai/docs/supported-models/